src/main/java/core/basesyntax/services/DataConverterImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.services;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < data.size(); i++) {

```

Comment:
Avoid hardcoding the index '1' to skip the header row. You might want to define a constant for the header index or document the assumption clearly.